Library/Koala/Koala.swift

```diff
@@ -1769,14 +1774,14 @@ private func properties(category category: KsApi.Category) -> [String:AnyObject]
     .withAllValuesFrom(parentProperties.prefixedKeys("parent_"))
 }
 
-private func properties(shareContext shareContext: ShareContext,
-                                     loggedInUser: User?,
-                                     shareActivityType: String? = nil) -> [String:AnyObject] {
+private func properties(shareContext: ShareContext,
+                        loggedInUser: User?,
+                        shareActivityType: UIActivityType? = nil) -> [String:Any] {
```

Comment:
oh wait, i take that back, `UIActivityType` is not an enum but one of them strange `struct`s that are also   `RawRepresentable` 🤔 . still tho, you can mostly handle as an enum with just a few small quirks.

part of me thinks that type would be better as an enum with a catch all `.custom(String)` case, but perhaps i'm missing something deeper.